content/en/blog/2024/otel-arrow-production/index.md

```diff
@@ -0,0 +1,413 @@
+---
+title: OpenTelemetry Protocol with Apache Arrow in Production
+linkTitle: OpenTelemetry Arrow in Production
+date: 2024-09-13
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Joshua MacDonald](https://github.com/jmacd) (ServiceNow, Inc), [Laurent
+  Querel](https://github.com/lquerel) (F5, Inc)
+cSpell:ignore: Querel Zstd
+draft: true
+issue: 5193
+sig: OpenTelemetry Arrow
+---
+
+## OpenTelemetry Protocol with Apache Arrow in production
+
+The OpenTelemetry Protocol with Apache Arrow (OTel-Arrow) project's exporter and
+receiver components for the OpenTelemetry Collector are now included in
+OpenTelemetry Collector-Contrib releases. This is a case study of our experience
+deploying OpenTelemetry Collectors using OTel-Arrow components as the primary
+ingestion path for internal telemetry data at ServiceNow Cloud Observability.
+
+Since the project’s announcement, our collaboration with F5, Inc. has focused on
```

Comment:
```suggestion
Since the project’s announcement, our collaboration with F5, Inc. has focused on
```

I struggle with the "our collaboration with F5, Inc.", this sounds like contributors from F5 (@lquerel?) are not part of the otel community or that there is an exclusive collaboration partnership between the otel project and F5, but since the code has been donated it is maintained and contributed to by OpenTelemetry community members who work for F5 (and other companies). If it is important from a branding perspective, we could say something like

```
Since F5 donated the OTel-Arrow code to the OpenTelemetry project, our work has focused on ...
```

Reply:
I feel it's important to F5, especially, to recognize this contribution and the substantial amount of effort that was invested before the donation was accepted. I've rephrased this as:

> Since the initial donation by F5, Inc. to the OpenTelemetry project, our efforts have focused on ...

hope that works, and I'm open to further edits.

Reply:
How about this slightly different formulation?

Since F5, Inc.’s initial contribution to the OpenTelemetry project, community members, including those from F5, have been collaboratively working to transform the OTel-Arrow exporter and receiver components into a reliable, high-performance solution for transporting bulk OpenTelemetry data across costly network links in real production environments.

Reply:
I like both variants, thanks! Add the one you prefer

Reply:
I used @lquerel's version.